js/base.js

```diff
@@ -39,7 +40,9 @@ export class API extends mix(BaseAPI).with(
         const auth = options.kwargs.auth === undefined ? true : options.kwargs.auth;
         delete options.kwargs.auth;
         if (auth) {
-            options.headers.Authorization = `Bearer ${this.token}`;
+            if (this.token) options.headers.Authorization = `Bearer ${this.token}`;
+            if (this.tokenTwitch)
+                { options.headers.AuthorizationTwitch = `Bearer ${this.tokenTwitch}`; }

```

Comment:
This layout that pretty generates sucks, go with:

```suggestion
            if (this.tokenTwitch) {
                options.headers.AuthorizationTwitch = `Bearer ${this.tokenTwitch}`;
            }
```